### PR TITLE
Leadership Page Updates (Fixes #12892)

### LINKED ISSUE #12892: Leadership Page Updates
## Description

Updates are outlined here:

https://docs.google.com/document/d/1eAFENW27zehTyuy6kVQHbRTDNe5W5Hk7x2hY5uYlj08/edit?usp=sharing

### PR DESCRIPTION
**Pull request by nathan-barrett**
## One-line summary

leadership page  updates

## Issue / Bugzilla link
#12892 

## Testing
http://localhost:8000/en-US/about/leadership/boards-of-directors/

### CODE REVIEW

**Review by craigcook — APPROVED** [review]
Looks like we weren't given an additional title for Kerry but we'll see if MJ has one we can add. Otherwise it's fine as-is (we could always add it later, too). So r+wc, just waiting for the official go-ahead before we merge.

**Comment by nathan-barrett** [comment]
Waiting for the go-ahead before we merge this

**Pull request merged**
